### DISCUSSION TITLE
我們與metaverse 的距離

### DISCUSSION THREAD

**Discussion by 109302003** [1101-GS4719-C/Python-Community-GS4719-C#145] [General]
[https://www.bnext.com.tw/article/64465/everything-you-need-to-know-about-metaverse-2]() Metaverse 無疑是今年討論度最高的關鍵字，現今VR AR發展越來越成熟，但實現虛擬世界可能還需很長一段時間。

**Comment by Jie2222** [comment]
的確還需要長時間，感覺可能會先出現很多虛擬養老院

**Comment by enoch-lightning** [comment]
或許以後大學的上課模式，有兩種方式可以選擇。第一種是到實體教室上課，第二種是，在其他地方，也能以擴增實境，也就是虛擬實境的方式上課。

**Comment by Tomlord1122** [comment]
元宇宙我覺得還要蠻長的一段時間來發展

**Comment by Isongzhe** [comment]
我個人覺得 元宇宙大概率是個噱頭。不過 近年來AR VR的成長確實是有目共睹的
